### ISSUE TITLE
Feature-Request: MultiAccount-support

### ISSUE THREAD

**Issue by Cyborgscode** [SchildiChat/schildichat-desktop#48]
I like to request Multi-Account-Support for Schildi.

At first, it would be enough to have a selectbox to choose the "active" connection. the "others" can be disconnected. 
This should be easy to add to the ui logic. 

Switch: Disconnect active useraccount -> switch db -> connect selected useraccount. 

Later, multiply active accounts  should be possible.

**Comment by su-ex** [comment]
Yes, multiple accounts would be great, but I won't implement that, because it's probably not that easy as you think, otherwise upstream Element would already have it. But if you think you can and want to implement it: contributions welcome!

**Comment by HKalbasi** [comment]
Upstream issue: https://github.com/vector-im/element-web/issues/2320

**Comment by RedSnt** [comment]
For some reason NeoChat supports switching accounts seamlessly, but it's just too bare boned - I wish this feature was available for more clients in general, but in particular SchildiChat.

Btw. the discussion has moved to: https://github.com/element-hq/element-meta/discussions/1681
